io-engine/src/lvs/lvs_store.rs

```diff
@@ -52,6 +52,16 @@ use crate::{
     pool_backend::PoolArgs,
 };
 
+use events_api::event::{
```

Comment:
Probably, we should consider moving mbus-specific code (dependent on generated code) to a separate module, like we do for gRPC-specific?

Reply:
Done